### PR TITLE
ROC-4494 Store additional information dring  court determination

### PR DESCRIPTION
**Pull request by dharmendrak**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ROC-4494

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/cmc/claimstore/services/FormaliseResponseAcceptanceServiceTest.java [hunk 1]
```diff
@@ -384,9 +359,9 @@ public void formaliseSettlementWithCourtDeterminedPaymentIntention() {
             .courtDetermination(CourtDetermination
                 .builder()
                 .courtDecision(paymentIntention)
+                .decisionType(COURT)
```
Comment (sabahirfan) [inline src/test/java/uk/gov/hmcts/cmc/claimstore/services/FormaliseResponseAcceptanceServiceTest.java]: would be nice to have tests for the scenarios when decision type is CLAIMANT/DEFENDANT after determination

File: domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/claimantresponse/ClaimantResponseType.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    ACCEPTATION("acceptation"),
+    REJECTION("rejection");
+
+    String description;
+
+    ClaimantResponseType(String description) {
+        this.description = description;
+    }
+
+    public String getDescription() {
```
Comment (sabahirfan) [inline domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/claimantresponse/ClaimantResponseType.java]: can we please use `@Getter` from lombok to be consistent

File: domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/claimantresponse/CourtDetermination.java [hunk 3]
```diff
@@ -21,19 +21,31 @@
...
+    @Valid
+    private final PaymentIntention courtPaymentIntention;
+
     private final String rejectionReason;
 
     @NotNull
     private final BigDecimal disposableIncome;
 
+    @NotNull
+    private final DecisionType decisionType;
```
[the remark this answers is not in the document]
Comment (dharmendrak) [reply domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/claimantresponse/CourtDetermination.java]: yes we are passing it from frontend as in the model

File: domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/CountyCourtJudgment.java [hunk 4]
```diff
@@ -41,31 +43,31 @@
...
     @Builder
     public CountyCourtJudgment(
         LocalDate defendantDateOfBirth,
         PaymentOption paymentOption,
         BigDecimal paidAmount,
         RepaymentPlan repaymentPlan,
         LocalDate payBySetDate,
-        StatementOfTruth statementOfTruth
+        StatementOfTruth statementOfTruth,
+        CountyCourtJudgmentType ccjType
```
Comment (sabahirfan) [inline domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/CountyCourtJudgment.java]: What about the existing `CountyCourtJudgment` data in `claim-store` database? Migration scripts?

Comment (dharmendrak) [reply domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/CountyCourtJudgment.java]: added

Comment (sabahirfan) [reply domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/CountyCourtJudgment.java]: Great thanks

**Pull request merged**